lib/solid/profile.js

```diff
@@ -176,6 +194,52 @@ SolidProfile.prototype.appendFromGraph =
   }
 
 /**
+ * Returns the value of a given "field" (predicate) from the profile's parsed
+ * graph. Note: If there are more than one matches for this predicate, .find()
+ * returns the first one.
+ * Usage:
+ *
+ *   ```
+ *   var inboxUrl = profile.find(vocab.solid('inbox'))
+ *   ```
+ * @method find
+ * @param predicate {NamedNode} RDF named node of the predicate
+ * @return {String} String value (or uri)
+ */
+SolidProfile.prototype.find = function find (predicate) {
```

Comment:
Could this be implemented on top of `SolidProfile.prototype.findAll` (by grabbing the first result)?